superset/assets/src/dashboard/util/publishSubscriberUtil.js

```diff
@@ -30,17 +32,71 @@ export function getUniqueActionsForSlice(slice) {
 export function getModalSliceIDFor(publishSubscriberMap, publisherId) {
     if (publishSubscriberMap && publishSubscriberMap.hasOwnProperty('subscribers')) {
         const subscribers = Object.values(publishSubscriberMap.subscribers);
-        var item = _.find(subscribers, function (item) {
+        var item = find(subscribers, function (item) {
             return ((item.actions.indexOf(USE_AS_MODAL) > -1) && (Object.keys(item.linked_slices).indexOf(publisherId) != -1));
         });
         return item ? item.id : undefined;
     }
     return undefined
 }
 
+export function getSubHeaderForSlice(subscribers, chartId, filters) {
+    let subHeader = '';
+    if (chartId != -1 && subscribers && subscribers[chartId]) {
+        const subscriber = subscribers[chartId];
+        if (subscriber.actions.indexOf(INCLUDE_IN_TITLE) > -1) {
+            for (let lsKey in subscriber.linked_slices) {
+                if (keyExists(lsKey, filters)) {
+                    subscriber.linked_slices[lsKey].forEach(linkedSlice => {
+                        if (linkedSlice.actions.indexOf(INCLUDE_IN_TITLE) > -1) {
+                            let columnName = linkedSlice['col'];
+                            let subHeaderValues = filters[lsKey][columnName];
+                            const values = subHeaderValues ? getSubHeaderValues(subHeaderValues) : [];
+
+                            if (subHeaderValues) {
+                                (subHeader != '') ? values.push(subHeader) : [];
```

Comment:
we should not use (ternary) operator if else condition is not useful...

We can change ternary operator like below
if (values.length > 0) {
                               subHeader =  (subHeader != '') ? values.push(subHeader).join(' | ') : values.join(' | ');
                            }

Reply:
Not readable for me, Just another way to deal with the problem, nothing wrong in the original one too.

Reply:
As per my view, Use ternary operators to set a value to a variable.

“Ternary operators vs if-else statements” by jraleman https://link.medium.com/mXsYlcAwuX

Reply:
You put up a wrong condition though, but you convinced me though so changing it the other way keeping it simple